os-daq-modules/daq_rswitch_offload.c

```diff
@@ -385,7 +383,7 @@ static int __rtnl_talk_iov(struct rtnl_handle *rtnl, struct iovec *iov,
 
 	status = sendmsg(rtnl->fd, &msg, 0);
 	if (status < 0) {
-		perror("Cannot talk to rtnetlink");
+		fprintf(stderr, "Cannot talk to rtnetlink");
```

Comment:
Could you please add a newline symbol ("\n") after switching from perror to fprintf?